index.d.ts

```diff
@@ -1,5 +1,23 @@
 declare module '@testcafe-community/axe' {
-  import { ElementContext, RunOnly, AxeResults, Result } from 'axe-core';
+  import { ElementContext, RunOnly, AxeResults, Result, Spec } from 'axe-core';
+
+  interface axeCheckI {
```

Comment:
Let's capitalize the name and drop the I at the end please

Reply:
Got it! Modifying.